test/parallel/test-fs-write-file-invalid-path.js

```diff
@@ -0,0 +1,76 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+const isWindows = process.platform === 'win32';
+
+if (!isWindows) return;
+
+const OK = [
+  'c:__test1',
+  'c:\\__test2',
+  '\\\\?\\c:\\__test4'
+];
+
+const FAIL = [
+  'c:__test:test2',
+  ':__test',
+  '1: __test',
+  '\\\\?\\UNC\\var\\__test:'
+];
+
+OK.forEach((path) => {
+  fs.writeFileSync(path, 'hello', { encoding: 'utf8' });
+  assert.strictEqual(fs.readFileSync(path, { encoding: 'utf8' }), 'hello');
+});
+
+FAIL.forEach((path) => {
+  assert.throws(() => {
+    fs.writeFileSync(path, 'hello', { encoding: 'utf8' });
+  }, common.expectsError({code: 'ERR_INVALID_WIN32_PATH', type: Error}), path);
+});
+
+OK.forEach((path) => {
+  fs.writeFile(path, 'world', { encoding: 'utf8' }, function(err) {
```

Comment:
Can you add `common.mustCall()` to the `readFile()` and `writeFile()` callbacks in this file.